src/schemathesis/cli/__init__.py

```diff
@@ -129,6 +130,7 @@ def run(  # pylint: disable=too-many-arguments
     auth_type: str,
     headers: Dict[str, str],
     checks: Iterable[str] = DEFAULT_CHECKS_NAMES,
+    exitfirst: bool = False,
```

Comment:
you could provide the third argument to `click.option` as `"exit_first"` and then it will be named  here as `exit_first` - in this case we won't have different names for this entity